README.md

```diff
@@ -141,7 +142,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 | [K](#k) | [L](#l) | [M](#m) | [N](#n) | [O](#o) |
 | [P](#p) | [Q](#q) | [R](#r) | [S](#s) | [T](#t) |
 | [U](#u) | [V](#v) | [W](#w) | [X](#x) | [Y](#y) |
-|         |         | [Z](#z) | 
+|         |         | [Z](#z) |
```

Comment:
Note for future reviewers: I don't know why we have a trailing space if it doesn't break anything 😅 I'd merge this because we do not want any of those to satisfy our code style/markdownlint/other formatters